WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationsViewController.swift

```diff
@@ -1658,7 +1691,6 @@ private extension NotificationsViewController {
         }
 
         static let sortKey = "timestamp"
-        static let allFilters = [Filter.none, .unread, .comment, .follow, .like]
```

Comment:
This had only callsite and I thought CaseIterable does exactly this. I can revert the change if there are concerns.

Reply:
Oh nice one!